File: backend spring/src/main/java/com/bloodbank/dto/request/HospitalRegisterRequest.java
package com.bloodbank.dto.request;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class HospitalRegisterRequest {
    private String hospitalName;
    private String email;
    private String password;
    private String address;
    private String licenseNumber;
    private String contactPerson;
}

File: BackendDotNet/DTOs/Request/HospitalRegisterRequest.cs
namespace BackendDotNet.DTOs.Request
{
    public class HospitalRegisterRequest
    {
        public string HospitalName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public string Address { get; set; } = string.Empty;
        public string LicenseNumber { get; set; } = string.Empty;
        public string ContactPerson { get; set; } = string.Empty;
    }
}
